### ISSUE TITLE
New Category: <Profession Knowledge Items>

### ISSUE THREAD

**Issue by rainkius** [Zottelchens-WoW-Addons/AdiBags-Dragonflight#4]
### Category Name

Profession Knowledge

### In-Game Color

#33937F

### In-Game Description

Items that provide profession knowledge

### README.md Description

Profession Knowledge items - used for Dragonflight's new profession specializations

### Item IDs

```Shell
198608
198682
201281
201279
201019
201283
201284
193909
201287
201285
201286
201288
201007
201280
201004
201282
198836
198606
201014
198710
193907
198687
198697
201015
198664
198971
201005
198658
198690
198680
198963
201006
198972
194697
198454
194702
198510
194704
194699
194703
194700
194708
194698
192132
200678
200677
194054
194041
200974
200972
198693
200976
200977
200980
200978
200979
200981
200973
200975
198974
192443
202011
198964
201012
198610
198611
198694
193902
198976
198599
193891
194045
194047
194046
194042
194044
194043
201013
201009
198712
198798
198659
198799
198800
198660
198686
198663
201003
201358
201359
201356
201357
201360
194063
201016
194039
199115
198669
201017
198973
198970
202014
198789
194064
198662
201301
201300
193905
198702
198612
198969
198675
198613
198670
198684
199122
193910
198966
201008
194081
198699
201716
201712
201709
201700
201715
201708
201705
201706
201717
201713
201710
201711
198692
198977
198975
198656
194080
194078
193904
198711
193913
193901
198968
193899
198965
198967
194055
193900
198519
198518
198520
198521
198522
198523
198524
198525
198526
198528
198704
194079
201010
201270
201268
201272
201273
201276
201274
201275
201277
201269
201271
193897
198607
198703
201020
198667
201011
198689
198978
194061
198609
198683
193898
194062
192131
193903
198685
201018
198696
198156
```

**Comment by rainkius** [comment]
Wowhead: Items > Misc > Other: Added in Expansion Dragonflight + Requires a Profession


Thank you again :)

**Comment by Zottelchen** [comment]
Thank you so much for sharing this list! It is implemented and will appear in the version 1.1.6 soon! :)
